test/models/resource.js

```diff
@@ -4,7 +4,7 @@ const models = require('../../models');
 
 describe('models.Resource', () => {
   beforeEach(async () => {
-    await helper.loadFixtures(['resources']);
+    await helper.loadFixtures(['categories', 'resources']);
```

Comment:
Now, because of REFERENTIAL INTEGRITY, you must load the categories fixtures BEFORE the resources fixtures, because the CategoryId values in the Resource records MUST match an existing id in the Categories table.